format-specs/geoparquet.md

```diff
@@ -1,5 +1,47 @@
+# Geospatial Parquet format
 
-## A Geospatial Parquet format
+## Overview
 
-This page is a scratch pad for ideas to define a 'geo-parquet' format, aligning on a default schema and format for geospatial data in Parquet.
+The [Apache Parquet][parquet] provides a standardized open-source columnar storage format. This specification defines how geospatial data
+should be stored in parquet format, including the representation of geometries and the required additional metadata.
 
+## Geometry columns
+
+Geometry columns are stored using the `BYTE_ARRAY` parquet type. They are encoded as WKB.
+
+## Metadata
+
+geoparquet files include additional metadata at two levels:
+
+1. File metadata indicating things like the version of this specification used
+2. Column (chunk) metadata with additional metadata for each geometry column
+
+## File metadata
+
+|     Field Name     |  Type  |                             Description                              |
+| ------------------ | ------ | -------------------------------------------------------------------- |
+| geoparquet_version | string | **REQUIRED** The version of the metadata standard used when writing. |
+| primary_column     | string | The "primary" geometry column.                                       |
```

Comment:
I'm not sure if this is appropriate to include or not.

Systems like geopandas (and maybe PostGIS?) can store multiple geometry dtype columns, but has the idea of an "active" or "primary" geometry that's used by default in geospatial operations.

If we include it, we'll need to update the description / add a section explaining it.

Reply:
I think it makes sense to have. I'm actually not sure of the precedence in different formats, but I believe most OGC api's/services allow multiple geometries, and also have operations that imply a 'default' geometry column. So definitely seems useful to me for the format to indicate what the default column is, especially to be used in a 'cloud-native' / data at-rest type of way.